package.json

```diff
@@ -41,8 +41,7 @@
     "fountainhead-core": "0.0.4",
     "iconv-lite": "^0.4.24",
     "ip": "^1.1.5",
-    "level": "^4.0.0",
-    "leveldown": "^4.0.1",
+    "level": "^5.0.1",
```

Comment:
This seems to be a safe update. See https://github.com/Level/level/blob/master/UPGRADING.md#v5